Recipes/Recipes/Resources/MainViewController.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  MainViewController.swift
+//  Recipes
+//
+//  Created by Keri Levesque on 2/10/20.
+//  Copyright © 2020 Lambda Inc. All rights reserved.
+//
+
+import UIKit
+
+class MainViewController: UIViewController {
+    // MARK: Properties
```

Comment:
Love these marks! Please keep up with doing them in ALL of your files. This will help with organization and help when others need to look over your code.